Taskfile.yml

```diff
@@ -393,6 +395,26 @@ tasks:
       - find "config/crd/out" -type f -delete
       - kustomize build config/crd > config/crd/out/azureserviceoperator.yaml
 
+  controller:gen-helm-manifest:
+    desc: Generate helm manifest using Kustomize for the release
+    dir: "v2/"
+    cmds:
+     - sed -i '' 's@\({{.PUBLIC_REGISTRY}}\)\(.*\)@\1{{.CONTROLLER_DOCKER_IMAGE}}@g' ./charts/azure-service-operator/values.yaml
```

Comment:
Does it make sense to put this stuff into a script in `scripts` and pass it the parameters it needs from here? So this would become something like:
`scripts/generate-helm-manifest.sh {{.KUBE_RBAC_PROXY}} {{.VERSION}} ...`?

The advantage I see there is that you can control some error behavior better and do stuff like `set -e` to ensure that failures in these commands trigger a failure in the `task`.

Reply:
I agree with you, I've made the changes. Thanks.